megalista_dataflow/uploaders/google_ads/conversions/google_ads_offline_conversions_uploader.py

```diff
@@ -106,12 +106,19 @@ def process(self, batch: Batch, **kwargs):
 
   def _do_upload(self, oc_service, execution, conversion_resource_name, customer_id, rows):
     logging.getLogger(_DEFAULT_LOGGER).info(f'Uploading {len(rows)} offline conversions on {conversion_resource_name} to Google Ads.')
-    conversions = [{
-          'conversion_action': conversion_resource_name,
-          'conversion_date_time': utils.format_date(conversion['time']),
-          'conversion_value': float(str(conversion['amount'])),
-          'gclid': conversion['gclid']
-    } for conversion in rows]
+    conversions = []
+    for row in rows:
+      conversion = {}
```

Comment:
I think you could start the conversion variable with the attributes of the lines 112, 113, 114 and 115 instead of defining them individually.

Reply:
done!